### PR TITLE
[Synfig Studio] app.cpp : some cleanup

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

**Comment by rodolforg** [comment]
Squash these commits? Split in different PR? :P

**Comment by ice0** [comment]
> Squash these commits? Split in different PR? :P

Don't squash )

**Comment by rodolforg** [comment]
Ok, I've finished it then.

**Comment by rodolforg** [comment]
Unless I should split this PR :P

**Review by ice0 — DISMISSED** [review]

File: synfig-studio/src/gui/app.cpp [hunk 1]
```diff
@@ -35,137 +35,123 @@
 #	include <config.h>
 #endif
 
-#include <fstream>
 #include <iostream>
-#include <locale>
-#include <cstring>
+#include <fstream>
```
Comment (rodolforg) [inline synfig-studio/src/gui/app.cpp:39]: Whoops! I noticed I sort it wrong here :P 

@ice0 did you change anything on this PR? Or i can fix it here and force-push it again?

Comment (ice0) [reply synfig-studio/src/gui/app.cpp:39]: No, I just rebased it for merge, but there were problems with Appveyor yesterday, so it still hasn't merged.

Comment (rodolforg) [reply synfig-studio/src/gui/app.cpp:38]: Ok. So I force pushed to sort it right

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**